WcaOnRails/app/views/competitions/index.html.erb

```diff
@@ -87,12 +89,11 @@ $(".js-scroll-to").click(function(e) {
   if(!$(this).hasClass('disabled')) {
     // Little hack to activate the affix thing and
     // move correctly to the anchor.
-    $("#simple-nav").removeClass('affix-top').addClass('affix');
```

Comment:
It's safe to just remove this without even calling that recalculate position method?